### DISCUSSION TITLE
How can I export notes from parent level without having to click on each not to export?

### DISCUSSION THREAD

**Discussion by T-Bolt94** [windingwind/zotero-better-notes#1204] [Q&A]
I want to export multiple notes from 100+ parent items but when I select all there is no export note option?
Thanks in advance for your help.

**Comment by windingwind** [comment]
Make a saved search for your notes and select all notes in that search.

**Reply by T-Bolt94** [reply]
Thank you very much for your help!

**Reply by T-Bolt94** [reply]
If I do this with CMD+A it selectes the parent items also and BetterNotes wont give the option to export notes. Do I have to click all of the notes individually?

**Comment by T-Bolt94** [comment]
![Screenshot 2024-11-19 at 08 43 21](https://github.com/user-attachments/assets/71b5831a-bcf4-48e9-ada3-738b3351107d)
![Screenshot 2024-11-19 at 08 43 44](https://github.com/user-attachments/assets/efcb9d27-4a3c-427f-bdee-e58d23d70f9a)

Here is the saved search with the output. Thanks!
